.pep8

```diff
@@ -0,0 +1,3 @@
+[pep8]
+ignore = E226,E302,E41
```

Comment:
Might change these in CI branch, seems we need to ignore a lot more if we don't want 7000+ errors